src/components/video/Video.styles.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components'
+import { colors } from '../../theme/colors'
+
+export const VideoContainer = styled.div`
+  position: relative;
+  padding-bottom: 56.25%;
+  margin-bottom: 2em;
+
+  iframe {
+    position: absolute;
+    height: 100%;
+    width: 100%;
+  }
+`
+
+export const StyledFigure = styled.figure`
+  margin: 0 0 2em 0;
+  cursor: pointer;
```

Comment:
This should only be applied to the `Button` and `Image` elements. It's currently being applied to the entire component which makes the `caption` look clickable.